tests/conftest.py

```diff
@@ -199,7 +199,7 @@ def enable_ssh_timout(dut):
     time.sleep(5)
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="session")
```

Comment:
Does this work? I tried to do this for my test reporting change and it complained because `ansible_adhoc` is module scoped.

Reply:
Ah wait I see your other change now.